src/common/pages/entry.tsx

```diff
@@ -159,7 +159,12 @@ class EntryPage extends BaseComponent<Props, State> {
     let replyDraft = ss.get(`reply_draft_${entry?.author}_${entry?.permlink}`);
     replyDraft = (replyDraft && replyDraft.trim()) || "";
 
-    this.setState({ isMounted: true, selection: replyDraft });
+    const wordsWithoutSpace: any = entry?.body.trim()?.split(/\s+/);
```

Comment:
this will crash the app when entry.body is undefined, that's why it was moved before, instead we have to make sure it renders only after it has entry.body perhaps...

Reply:
Noted,  I'll move to ensure and rewrite logic